packages/osd-dev-utils/src/osd_client/osd_client_requester.ts

```diff
@@ -37,7 +37,7 @@ import { ToolingLog } from '../tooling_log';
 
 const isConcliftOnGetError = (error: any) => {
   return (
-    isAxiosResponseError(error) && error.config.method === 'GET' && error.response.status === 409
+    isAxiosResponseError(error) && error.config?.method === 'GET' && error.response.status === 409
```

Comment:
Would like some comment here, is there any breaking change in axios in v0.28 that it may not bring config when throw error?

Reply:
[Their changelog](https://github.com/axios/axios/releases?q=v0.28.0&expanded=true) doesn't document any breaking changes; this is more than likely just fine tuning of the types def.